auto_round/autoround.py

```diff
@@ -497,7 +508,7 @@ def quantize_and_save(self, output_dir: str = "tmp_autoround", format: str = "au
                 if (self.sym and ("gptq" not in format and "awq" not in format)) or self.bits == 3:
                     format = format.replace('auto_round', 'auto_round:auto_gptq')
                     formats[index] = format
-                if self.bits == 4 and not self.sym:
+                if self.bits == 4 and not self.sym and ("gptq" not in format and "awq" not in format):
```

Comment:
just format=="auto_round", we may have auto_round:triton or others later  
help fix this in the above code " if (self.sym and ("gptq" not in format and "awq" not in format))"